### PR TITLE
9.0 mig runbot website display

### PR DESCRIPTION
**Pull request by jjscarafia**

### CODE REVIEW

**Comment by jjscarafia** [comment]
Lint error if because of travis2docker, PR to fix that here https://github.com/OCA/runbot-addons/pull/100

**Comment by moylop260** [comment]
👍

**Comment by pedrobaeza** [comment]
Rebase to have clean Travis.

File: runbot_website_display/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline runbot_website_display/README.rst]: Put license link like in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: runbot_website_display/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+In the tree view, you can also drag repos to reorder them on the website.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/146/9.0
+
+For further information, please visit:
```
Comment (pedrobaeza) [inline runbot_website_display/README.rst]: Remove this link

File: runbot_website_display/README.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/runbot-addons/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/runbot-addons/issues/new?body=module:%20runbot_website_display%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline runbot_website_display/README.rst]: Adapt this to the new format

**Review by pedrobaeza — APPROVED** [review]
Minor nitpickings, but not blocking

**Comment by jjscarafia** [comment]
@pedrobaeza I think now it is ok, sorry for many commits, don't know why but rebase break the push so I need a merge later

**Comment by pedrobaeza** [comment]
Not really, because there is some mismatch in current commits.

**Comment by jjscarafia** [comment]
@pedrobaeza any suggestion? Perhups better I make a new clean PR

**Comment by pedrobaeza** [comment]
What you can do locally:
- Start from origin/9.0 branch
- checkout -b for creating a new branch from it with a temp name.
- cherry-pick your commit(s)
- delete branch 9.0-mig-runbot_website_display
- do `git checkout -b 9.0-mig-runbot_website_display` from your branch.
- add remote if not yet present: `git remote add adhoc-dev ...`
- push force to GitHub: `git push adhoc-dev 9.0-mig-runbot_website_display -f`

This way, you can reuse this PR.

**Pull request merged**